contentsquare/src/main/java/com/tealium/remotecommands/contentsquare/ContentsquareInstance.kt

```diff
@@ -42,6 +44,33 @@ class ContentsquareInstance(private val application: Application) : Contentsquar
         }
     }
 
+    override fun sendUserIdentifier(userId: String) {
+        Log.d(TAG, "Sending user identifier: $userId")
+        Contentsquare.sendUserIdentifier(userId)
+    }
+
+    override fun sendCustomVars(screenName: String, customVars: Array<JSONObject>) {
+        Log.d(TAG, "Sending custom variables for screen: $screenName")
+        val csCustomVars = customVars.mapNotNull { json ->
+            val index = json.optInt(CustomVars.INDEX, -1)
+            val name = json.optString(CustomVars.NAME)
+            val value = json.optString(CustomVars.VALUE)
+
+            if (index in 1..20) {
```

Comment:
saw that iOS was recommending not being so restrictive on this too, which seems fair.
perhaps when converting this, we can simply fail early when we get the index:
```kotlin
 val index = json.optInt(CustomVars.INDEX, -1)
 if (index <= 0) return@mapNotNull null
 // ...
```